### PR TITLE
Updated k8s charts

### PR DESCRIPTION
**Pull request by pborreli**

### CODE REVIEW

File: api/Dockerfile [hunk 1]
```diff
@@ -13,6 +13,7 @@ RUN apk add --no-cache \
 		file \
 		gettext \
 		git \
+		postgresql-client \
```
Comment (dunglas) [inline api/Dockerfile]: to remove

File: api/docker/nginx/conf.d/default.conf [hunk 2]
```diff
@@ -13,6 +13,10 @@ server {
         #set $upstream_host php;
         #fastcgi_pass $upstream_host:9000;
 
+        # Bigger buffer size to handle cache invalidation headers expansion
```
Comment (dunglas) [inline api/docker/nginx/conf.d/default.conf]: To remove, specific to the demo

File: api/docker/varnish/conf/default.vcl [hunk 3]
```diff
@@ -29,11 +33,27 @@ sub vcl_backend_response {
   set beresp.grace = 1h;
 }
 
+sub vcl_recv {
+ if (req.http.X-Blackfire-Query) {
```
Comment (dunglas) [inline api/docker/varnish/conf/default.vcl]: To remove specific to Blackfire

File: api/helm/api/templates/secrets.yaml [hunk 4]
```diff
@@ -4,15 +4,17 @@ kind: Secret
...
+    app.kubernetes.io/name: {{ include "name" . }}
+    app.kubernetes.io/part-of: {{ include "name" . }}
+    helm.sh/chart: {{ include "chart" . }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
 type: Opaque
 data:
   {{ if .Values.postgresql.enabled }}
-  database-url: {{ printf "postgres://%s:%s@%s/%s" .Values.postgresql.postgresUser .Values.postgresql.postgresPassword $postgresqlServiceName .Values.postgresql.postgresDatabase | b64enc | quote }}
+  database-url: {{ printf "pgsql://%s:%s@%s/%s?serverVersion=9.6" .Values.postgresql.postgresqlUsername .Values.postgresql.postgresqlPassword $postgresqlServiceName .Values.postgresql.postgresqlDatabase | b64enc | quote }}
```
Comment (dunglas) [inline api/helm/api/templates/secrets.yaml]: We should keep `serverVersion` to prevent an extra SQL query at every HTTP request.

Comment (pborreli) [reply api/helm/api/templates/secrets.yaml]: we didn't remove it, so I guess it's good now

File: api/helm/api/.helmignore [hunk 5]
```diff
@@ -19,3 +19,4 @@
 .project
 .idea/
 *.tmproj
+.vscode/
```
Comment (vincentchalamon) [inline api/helm/api/.helmignore]: There are many other rules to add, you could have a look at a generic global .gitignore configuration: https://gist.github.com/vincentchalamon/d5defad563ed49d9306a4aa57dfd4498

Comment (pborreli) [reply api/helm/api/.helmignore]: yes not mine .. it was inside the demo PR

File: api/helm/api/values.yaml [hunk 6]
```diff
@@ -27,28 +33,51 @@ varnish:
...
+    # kubernetes.io/ingress.class: gce
+    # kubernetes.io/tls-acme: "true"
+  tls:
+    # Secrets must be manually created in the namespace, you can also use cert-manager.
+    # - hosts:
+      # - example.com
+      # - mercure.example.com
+  hosts:
+    api:
+      host: example.com
```
Comment (vincentchalamon) [inline api/helm/api/values.yaml]: `api.example.com` would be more efficient to prevent any conflict with mercure host

Comment (pborreli) [reply api/helm/api/values.yaml]: already discussed with @dunglas but OK to change it if needed

Comment (vincentchalamon) [reply api/helm/api/values.yaml]: What did @dunglas say about it?

Comment (pborreli) [reply api/helm/api/values.yaml]: I think we said that `*.example.com` is different and won't conflict than `example.com`

Comment (vincentchalamon) [reply api/helm/api/values.yaml]: :+1:

**Comment by soyuka** [comment]
thanks @pborreli !

**Pull request merged**